### ISSUE TITLE
develop version is not getting uploaded to bintray

### ISSUE THREAD

**Issue by jgebal** [utPLSQL/utPLSQL-cli#67]
After build on Travis, the develop branch is not getting pushed to bintray.
Outputs in Travis show:
```
[Bintray Upload] Reading descriptor file: bintray.json
[Bintray Upload] Uploading file 'dist/utPLSQL-cli-develop-201803132129.zip' to utPLSQL-cli-develop-201803132129.zip
[Bintray Upload] Bintray response: 401 Unauthorized. This resource requires authentication
[Bintray Upload] Publishing version 'develop' of package 'utPLSQL-cli-develop'...
[Bintray Upload] Bintray response: 401 Unauthorized. This resource requires authentication
```

https://travis-ci.org/utPLSQL/utPLSQL-cli/jobs/353061401#L1697

We're still using bintray account of @viniciusam - maybe it's time to move to utplsql account?

**Comment by jgebal** [comment]
Also, the readme.md in project still points to builds in:
https://bintray.com/viniciusam/utPLSQL-cli/utPLSQL-cli-develop#files

**Comment by pesse** [comment]
Closed via #73
